content/en/docs/setup/install/multicluster/simplified/index.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Simplified Multicluster Install (experimental)
+description: Configure an Istio mesh spanning multiple Kubernetes clusters.
+weight: 1
+keywords: [kubernetes,multicluster]
+---
+
+This guide describes how to configure an Istio mesh that includes multiple Kubernetes clusters using a simplified experimental approach.
+We hope to continue developing this functionality in coming releases, so we'd love your feedback on the overall flow.
+
+We focus here on the details of getting a multicluster mesh wired up, refer to [multicluster deployment model](/docs/setup/deployment-models/#multiple-clusters) for
+additional background information. We'll show how to connect two clusters that are on the same network together, and then
+how to attach a third cluster that's on a different network.
+
+## Before you begin
+
+The procedures we describe here are primarily intended to be used with relatively pristine clusters,
+where Istio hasn't already been deployed. TBD: WHAT SHOULD WE SAY ABOUT EXISTING CLUSTERS? SOL?
+
+For the sake of explanation, this task assumes you have created three Kubernetes clusters:
+
+- A cluster named `cluster-east-1` on the network named `network-east`.
+- A cluster named `cluster-east-2` on the network named `network-east`.
+- A cluster named `cluster-west-1` on the network named `network-west`.
+
+These clusters shouldn't have Istio on them yet. Take a look at the [platform setup instructions](/docs/setup/platform-setup)
+for any special instructions for your particular environment.
+
+## Initial preparations
+
+You need to do a few one-time steps in order to be able to setup a multicluster mesh:
+
+1. Decide on what the name of your multicluster mesh will be. Something short but memorable is your best choice here:
+
+    {{< text bash >}}
+    $ export MESH_ID=mymeshname
+    {{< /text >}}
+
+1. Decide on the organization name to use in the intermediate certificates created to let the clusters communicate with one
+another. This should generally be derived from your organization's DNS name:
+
+    {{< text bash >}}
+    $ export ORG_NAME=mymeshname.mycompanyname.com
+    {{< /text >}}
+
+1. Create a working directory where to store a number of files produced during the cluster
+onboarding process:
+
+    {{< text bash >}}
+    $ export WORKDIR=mydir
+    $ mkdir -p ${WORKDIR}
+    $ cd ${WORKDIR}
+    {{< /text >}}
+
+1. Download the setup script to your working directory:
+
+    {{< text bash >}}
+    $ curl {{< github_file >}}/samples/multicluster/setup-mesh >./setup-mesh.sh
+    $ chmod +x ./setup-mesh.sh
+    {{< /text >}}
+
+1. Create a new [Kubernetes configuration file](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/#define-clusters-users-and-contexts)
```

Comment:
kubeconfig are generally created/updated by the Kubernetes creation tool (e.g. GKE, minikube, kind). We've already presumed three clusters exist. Can we simply state that you need the KUBECONFIG and CONTEXT for each cluster?